### PR TITLE
Update get-snapshot-status.md

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
Get snapshot status docs have incorrect information.

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/6428

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@astute-decipher - Can you please review this for technical accuracy? Thanks so much.

**Comment by hdhalter** [comment]
@kkhatua - Can someone on your team sign off on this one?

File: _api-reference/snapshots/get-snapshot-status.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Three request variants provide flexibility:
 
 * `GET _snapshot/_status` returns the status of all currently running snapshots in all repositories.
 
-* `GET _snapshot/<repository>/_status` returns the status of only currently running snapshots in the specified repository. This is the preferred variant.
+* `GET _snapshot/<repository>/_status` returns all currently running snapshots in the specified repository. This is the preferred variant.
 
-* `GET _snapshot/<repository>/<snapshot>/_status` returns the status of all snapshots in the specified repository whether they are running or not.
+* `GET _snapshot/<repository>/<snapshot>/_status` returns detailed status information of a specific repository, regardless of whether it's currently running or not.
```
Comment (harishbhakuni) [inline _api-reference/snapshots/get-snapshot-status.md:34]: specific repository -> specific snapshot

File: _api-reference/snapshots/get-snapshot-status.md [hunk 2]
```diff
@@ -420,4 +420,4 @@ All property values are Integers.
...
+| shards | list of Objects | List of objects containing information about the shards that include the snapshot. Properies of the shards are listed below in bold text. <br /><br /> **stage**: Current state of shards in the snapshot. Shard states are: <br /><br /> * DONE: Number of shards in the snapshot that were successfully stored in the repository. <br /><br /> * FAILURE: Number of shards in the snapshot that were not successfully stored in the repository. <br /><br /> * FINALIZE: Number of shards in the snapshot that are in the finalizing stage of being stored in the repository. <br /><br />* INIT: Number of shards in the snapshot that are in the initializing stage of being stored in the repository.<br /><br />* STARTED:  Number of shards in the snapshot that are in the started stage of being stored in the repository.<br /><br /> **stats**: See [Snapshot file stats](#snapshot-file-stats). <br /><br /> **total**: Total number and size of files referenced by the snapshot. <br /><br /> **start_time_in_millis**: Time (in milliseconds) when snapshot creation began. <br /><br /> **time_in_millis**: Total time (in milliseconds) that the snapshot took to complete.  |
```
Comment (harishbhakuni) [inline _api-reference/snapshots/get-snapshot-status.md:423]: don't see any diff in this.

**Comment by hdhalter** [comment]
Thanks, @kkhatua and @harishbhakuni! I didn't see any change in that second paragraph either. Maybe it moved by a line? Do you approve?

**Review by kkhatua — APPROVED** [review]
Verified that the second change is basically a whitespace that is being picked up in the diff and can be ignored.
Approved the PR. Thanks @hdhalter

**Review by harishbhakuni — APPROVED** [review]

File: _api-reference/snapshots/get-snapshot-status.md [hunk 3]
```diff
@@ -29,9 +29,9 @@ Three request variants provide flexibility:
 
 * `GET _snapshot/_status` returns the status of all currently running snapshots in all repositories.
 
-* `GET _snapshot/<repository>/_status` returns the status of only currently running snapshots in the specified repository. This is the preferred variant.
+* `GET _snapshot/<repository>/_status` returns all currently running snapshots in the specified repository. This is the preferred variant.
```
Comment (Naarcha-AWS) [inline _api-reference/snapshots/get-snapshot-status.md:32]: Why is this the preferred variant?

Comment (Naarcha-AWS) [inline _api-reference/snapshots/get-snapshot-status.md:32]: Why is this the preferred variant of this command?

File: _api-reference/snapshots/get-snapshot-status.md (hunk 2 above)
Comment (Naarcha-AWS) [inline _api-reference/snapshots/get-snapshot-status.md:423]:
```suggestion
| shards | list of Objects | List of objects containing information about the shards that include the snapshot. OpenSearch returns the following properties about the shards. <br /><br /> **stage**: Current state of shards in the snapshot. Shard states are: <br /><br /> * DONE: Number of shards in the snapshot that were successfully stored in the repository. <br /><br /> * FAILURE: Number of shards in the snapshot that were not successfully stored in the repository. <br /><br /> * FINALIZE: Number of shards in the snapshot that are in the finalizing stage of being stored in the repository. <br /><br />* INIT: Number of shards in the snapshot that are in the initializing stage of being stored in the repository.<br /><br />* STARTED:  Number of shards in the snapshot that are in the started stage of being stored in the repository.<br /><br /> **stats**: See [Snapshot file stats](#snapshot-file-stats). <br /><br /> **total**: Total number and size of files referenced by the snapshot. <br /><br /> **start_time_in_millis**: Time (in milliseconds) when snapshot creation began. <br /><br /> **time_in_millis**: Total time (in milliseconds) that the snapshot took to complete.  |
```

**Review by Naarcha-AWS — APPROVED** [review]
One point of clarification and a fix of vale errors. Otherwise, LGTM

File: _api-reference/snapshots/get-snapshot-status.md (hunk 3 above)
Comment (hdhalter) [reply _api-reference/snapshots/get-snapshot-status.md:32]: @harishbhakuni  - do you know why this is the recommended option?

**Pull request merged**